### PR TITLE
tests: adding more logs to 20-cidr-limit

### PR DESCRIPTION
**Pull request by ianvernon**
Add more logs to 20-cidr-limit which has been failing sporadically in the build.

Signed-off by: Ian Vernon <ian@cilium.io>

Fixes: #2189

### CODE REVIEW

**Comment by aanm** [comment]
Do you wan't to merge it or try to retrigger the failure on this PR? I'm fine either way

**Review by aanm — APPROVED** [review]

**Comment by ianvernon** [comment]
@aanm I don't know how to trigger this issue - my idea is that once this is merged we can get more insight into why it's failing.

**Comment by ianvernon** [comment]
Please do not merge this yet - need to add one more thing to the test so we can actually figure out why the test is failing. For some reason `set +x` is ran right after `set -ex` is enabled, and we need the output from `set -x` for debugging failures in the cases where logs are not output.

EDIT: this is good to merge.

**Pull request merged**
